lib/screens/release_health/release_health_view_model.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:redux/redux.dart';
+import 'package:sentry_mobile/redux/actions.dart';
+import 'package:sentry_mobile/redux/state/app_state.dart';
+import 'package:sentry_mobile/types/organization.dart';
+import 'package:sentry_mobile/types/project.dart';
+import 'package:sentry_mobile/types/release.dart';
+
+class ReleaseHealthViewModel {
+  ReleaseHealthViewModel(this._store, this._organization, this.project, this.releases, this.loading);
+  final Store<AppState> _store;
+  final Organization _organization;
+
+  final Project project;
+  List<Release> releases;
+  bool loading;
+
+  static ReleaseHealthViewModel fromStore(Store<AppState> store) =>
```

Comment:
let's use `Factory` instead of static methods

Reply:
or named ctor would work too https://github.com/erluxman/awesomefluttertips/blob/master/page3.md#tip-51--named-constructor

Reply:
Read up on this in the dart doc, i guess the named constructor fits better, as we are not caching instances or doing anythong other besides initializing fields.